app/src/main/java/com/github/houseorganizer/houseorganizer/house/CreateHouseholdActivity.java

```diff
@@ -79,7 +177,7 @@ private void signalFailure(Exception exception, View v) {
         startActivity(intent);
     }
 
-    private Map<String, Object> createHousehold(CharSequence houseHoldName, int lat, int lon) {
+    private Map<String, Object> createHousehold(CharSequence houseHoldName, double lat, double lon) {
```

Comment:
That's better than integer coordinates 😆

Reply:
don't you like your position +- 100km ? xD